app/_hub/kong-inc/injection-protection/how-to/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+nav_title: Detect boolean-based blind SQL injection with a custom regex
+title: Detect boolean-based blind SQL injection with a custom regex
+---
+
+You can use the custom regex configuration of the Injection Protection plugin to detect additional regex patterns in injection attacks. 
+
+In this example, you'll configure the Injection Protection plugin to detect a boolean-based blind SQL injection in a URL using custom regex. This custom regex would detect injections in a URL like the following: `https://example.com/products?id=1'`
+
+## Set up the Injection Protection plugin
+
+Enable the Injection Protection plugin on a route.
+
+1. Create a service:
+    ```sh
+    curl -i -s -X POST http://localhost:8001/services \
+        --data name=ip_example_service \
+        --data url='http://httpbin.org'
+    ```
+
+2. Create a route:
+    ```sh
+    curl -i -X POST http://localhost:8001/services/ip_example_service/routes \
+        --data 'paths[]=/ip_mock' \
+        --data name=ip_route
+    ```
+
+3. Enable the Injection Protection plugin on the route:
```

Comment:
ChatGPT helped me with the config, but I tested it and it was enabled on the route successfully.